build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.3.0-beta2'
```

Comment:
didn't you say you were going to roll back on this? Am pretty uncomfortable moving onto the beta release, especially if we don't need to.